src/propagation.cpp

```diff
@@ -17,6 +19,12 @@ const std::string sampling_priority_header = "x-datadog-sampling-priority";
 // interop.
 const ot::string_view baggage_prefix = "ot-baggage-";
 
+// Key names for binary serialization in JSON
```

Comment:
Stick 'em in an anonymous namespace maybe

Reply:
They are, but github didn't show enough context.